app/views/Title.scala

```diff
@@ -18,13 +18,16 @@ package views
 
 import play.api.i18n.Messages
 
-case class Title(title: String)
```

Comment:
The advantage of getting rid of the 'string' constructor and apply method is that if the Title is (incorrectly) constructed with a translated message (e.g. `Title(messages("consignmentReferences.title"))`) it will still render the title correctly with the formatted "title - service - GOV.UK"